Automattic/jetpack
# Search: Add pricing and tier information to Plans page

Jetpack equivalent to Automattic/wp-calypso#40306.

#### Changes proposed in this Pull Request:
* Adds support for `/sites/<id>/products` API route via `<QuerySiteProducts />`, `site-products` state handling, and `jetpack/v4/site/products` local API route.
* Adds handling for showing Search plan tiers
* Adds handling for showing Search plan prices

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Yes, this adds support for showing Search plan tiers to the wp-admin Jetpack plans page. This is being merged into a feature branch, however.

#### Testing instructions:
1. You must apply Phabricator diff 153244 to your sandbox. This is a variant of D38944-code.

2. Apply changes from this PR to your Jetpack installation. If you're running a dockerized instance, you may need to [update your `extra_hosts`](https://docs.docker.com/compose/compose-file/#extra_hosts) to properly sandbox the WPCOM API.

3. Navigate to the Jetpack Plans page at `/wp-admin/admin.php?page=jetpack#/plans`. 

4. Ensure that both the record counts and the tier information render as expected.

<img width="1068" alt="Screen Shot 2020-03-25 at 6 10 37 PM" src="https://user-images.githubusercontent.com/4044428/77597305-0bde8800-6ec4-11ea-9fe6-8978890572e7.png">

#### Proposed changelog entry for your changes:
* None for now.
